packages/web/pages/user/my-account/curate-profile.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Protected } from '../../../components/Authorization';
+import { UserProfile } from '../../../components/UserProfile';
+import { CurateSpecialties, CurateFormSpecialties } from '../../../components/CurateSpecialties';
+import HeaderProfile from '../../../components/HeaderProfile';
+import { ROLES as rolesEnum } from '../../../utils/enums/api.enum';
+import { getReviewerUser } from '../../../services/user';
+
+const CurateProfile = ({ categories = [] }) => {
+	const normalizeCategories = categories
+		.map(
```

Comment:
```suggestion
		.map(
```

Aplique um filter antes do map, assim não precisará do check `category.parent_id &&`, também mova os `categories.filter` para variáveis isoladas, assim o código fica melhor de entender

`.filter(category => category.parent_id).map()`

Ao inves de colocar `categories.filter((c) => c.id === category.parent_id)[0].term`, é melhor salvar isso em uma variável, outra coisa que vi foi que seu filter, está sendo usado basicamente como um find, é melhor utilizar o `.find` (você fez isso em outra parte do código) ficando assim `const labelCategory = categories.find((innerCategory) => innerCategory.id === category.parent_id)`

Reply:
Feito!